src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -86,8 +99,64 @@
     useDefault,
 )
 from idaes.core.util.config import is_physical_parameter_block
+from idaes.core.initialization import ModularInitializerBase
 from idaes.models.unit_models.mscontactor import MSContactor
 
+
+class SolventExtractionInitializer(ModularInitializerBase):
+    """
+    This is a general purpose Initializer  for the Solvent Extraction unit model.
+
+    This routine calls the initializer for the internal MSContactor model.
+
+    """
+
+    CONFIG = ModularInitializerBase.CONFIG()
+
+    # CONFIG.declare(
```

Comment:
All this commented code should be removed - it is not applicable to modular initializers anyway.